docs/develop/01_sdk/02_cookbook/06_upgrading_to_v0_29/index.md

```diff
@@ -0,0 +1,13 @@
+---
+id: howto-upgrade-v29-index
+title: Upgrading to v0.29
+---
+
+Version 0.29.0 is the result of our efforts to make the sdk easier to understand and to use.
+
+As a consequence, quite a few things have changed relative to previous versions.
+These pages serve as a reference point for what to consider when upgrading to make your transition as smooth as possible.
+
+Find out what has changed and how to upgrade in the [release notes](https://github.com/KILTprotocol/sdk-js/releases/tag/0.29.0).
```

Comment:
Why doesn't this break the build step? It's 404, so it should be seen as broken link.

Reply:
I was asking myself the same thing. It does return a 404 so something is off.